### PR TITLE
Feature/secondary endpoint

### PR DESCRIPTION
**Pull request by sutjin**

### CODE REVIEW

File: src/loader.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
 const callbackName = 'recaptchaOnLoad';
 const RECAPTCHA_URL = `https://www.google.com/recaptcha/api.js?onload=${callbackName}&render=explicit`;
+const SECONDARY_URL = `https:/www.recaptcha.net/recaptcha/api.js?onload=${callbackName}&render=explicit`;
```
Comment (tr-lewis) [inline src/loader.js:3]: I think we are missing a slash here ```https:/www```

File: src/test/loader.test.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import Loader from '../loader';
 
 const callbackName = 'recaptchaOnLoad';
 const RECAPTCHA_URL = `https://www.google.com/recaptcha/api.js?onload=${callbackName}&render=explicit`;
+const SECONDARY_URL = `https:/www.recaptcha.net/recaptcha/api.js?onload=${callbackName}&render=explicit`;
```
Comment (tr-lewis) [inline src/test/loader.test.js:5]: I think we are missing a slash here ```https:/www```

File: src/loader.js (hunk 1 above)
Comment (sutjin) [reply src/loader.js:3]: good catch! chrome auto serolved the endpoint so i didnt see it. will push a minor fix

**Comment by sutjin** [comment]
@tr-lewis looks like issue comes when I converted the loader into a class. I have fixed it and will release it as a minor update

**Pull request merged**